erizo_controller/erizoJS/erizoJSController.js

```diff
@@ -337,26 +344,24 @@ exports.ErizoJSController = function (spec) {
     that.addSubscriber = function (from, to, options, callback) {
 
         if (publishers[to] === undefined){ 
-            log.warn("Trying to subscribe ", from, " to unavailable publisher", to);
+            log.warn("message: addSubscriber to unknown publisher, code: " + WARN_NOT_FOUND + ", streamId: " + to + ", clientId: " + from);
             //We may need to notify the clients 
             return;
         }
         if (subscribers[to][from] !== undefined){
-            log.warn("This subscription from", from, "to", to, "is already made, will remove it first and resubscribe");
+            log.warn("message: Duplicated subscription will resubscribe, code: " + WARN_CONFLICT + ", streamId: " + to + ", clientId: " + from);
             that.removeSubscriber(from,to);
         }
-
-        log.info("Adding subscriber from ", from, 'to ', to, 'audio', options.audio, 'video', options.video);
-
         var wrtcId = from+"_"+to;
-
+        log.info("message: Adding subscriber id: " + wrtcId + ", options:", options);
         var wrtc = new addon.WebRtcConnection(wrtcId, true, true, GLOBAL.config.erizo.stunserver, GLOBAL.config.erizo.stunport, GLOBAL.config.erizo.minport, GLOBAL.config.erizo.maxport,false,
                 GLOBAL.config.erizo.turnserver, GLOBAL.config.erizo.turnport, GLOBAL.config.erizo.turnusername, GLOBAL.config.erizo.turnpass);
 
+        wrtc.wrtcId = wrtcId;
         subscribers[to][from] = wrtc;
         publishers[to].muxer.addSubscriber(wrtc, from);
         wrtc.minVideoBW = publishers[to].minVideoBW;
-        log.debug("Setting scheme to sub WRTC", publishers[to].scheme);
+        log.debug("message: Setting scheme from publisher to subscriber id: " + wrtcId + ", scheme: " + publishers[to].scheme);
```

Comment:
`log.debug("message: Setting scheme from publisher to subscriber, id: " + wrtcId + ", scheme: " + publishers[to].scheme);`